package.json

```diff
@@ -41,16 +41,16 @@
         "@types/jest": "^26.0.24",
         "@types/react": "^17.0.0",
         "@types/react-test-renderer": "^17.0.0",
-        "@typescript-eslint/eslint-plugin": "^2.34.0",
-        "@typescript-eslint/parser": "^2.34.0",
+        "@typescript-eslint/eslint-plugin": "^4.28.3",
```

Comment:
Needed to update to get rid of this lint bug message:
![image](https://user-images.githubusercontent.com/13872908/134324947-5fe2a85d-b2e9-4ba2-9f62-5cc0f08908ec.png)